tests/integration/cases.yaml

```diff
@@ -273,3 +273,18 @@ test_cases:
     namespace: default
     expected_msg: pod/pod-poff created
     expected_result: null
+  certificate:
+  - id: cu
```

Comment:
that's the same id as cosign unsigned test case. let's use somethign different like `cert-u`. same below for the `cs` id -> `cert-s`